packages/upload-api/src/utils/rate-limits.js

```diff
@@ -0,0 +1,34 @@
+/**
+ *
+ * @param {import("../types").RateLimit} rateLimit
+ */
+const isBlock = (rateLimit) => rateLimit.rate === 0
+
+/**
+ *
+ * @param {import("../types").RateLimit[]} rateLimits
+ */
+const areAnyBlocks = (rateLimits) => rateLimits.some(isBlock)
```

Comment:
I would have called it `isBlocked`

Reply:
hm, this is a function which, given a list of rate limits, return true if any of them is a "block" - ie, if
any of them have a rate of 0 - I'll add a comment to that effect, but I like this name better for that